### DISCUSSION TITLE
Colores de Sitio Web

### DISCUSSION THREAD

**Discussion by rocappellano** [rpgrca/proyecto-ispc-fullstack#107] [Ideas]
¿Qué colores vamos a utilizar en el sitio?
Para colocar en el estilo de las páginas más tarde

**Comment by rpgrca** [comment]
Aunque parecemos una sucursal de Tarjeta Naranja no me disgusta del todo, al menos hasta ahora. En verdad en casa tengo dos oficinas y en ambas la pared sobre la que tengo la mesa y a las que miro están pintadas de naranja porque promueve la actividad mental, te hacen estar despierto (al contrario del verde que te calma). Pero el naranja es un color difícil de combinar, con verde queda bastante mal. Algunas combinaciones de colores: https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=F57F17

**Comment by claubott** [comment]
Hola gente! 

Yo me asigné la  página de contacto anónima y este es el estilo que tenía en mente:
(No le presten atención al logo ni a los items del menú, que no son los acordados.)

![Image](https://user-images.githubusercontent.com/37263110/194734618-7da58cf5-7732-48dd-8ffa-a578116412b8.jpg)

Mi inspiración fue esta plantilla gratuita de bootstrap 5: https://themewagon.com/themes/free-bootstrap-5-html5-restaurant-food-website-template-foodwagon/. Este sitio es un buen recurso, tiene un montón de plantillas gratuitas. 

El fondo naranja puede ser para un hero section y para lo demás se puede usar fondo blanco y un gris si necesitamos intercalar un cambio de color. Los colores de ese tema son estos:

![imagen](https://user-images.githubusercontent.com/37263110/194783452-8ff69b95-e158-4262-a128-f9c55c077f61.png)

También tengo esta alternativa de menú oscuro:

![Image](https://user-images.githubusercontent.com/37263110/194734622-e2fb63d3-4c66-44ba-9b78-578dd05a2f48.jpg)

**Comment by rpgrca** [comment]
Algo que no me gusta es que los íconos en ubicación son un rojo que no contrasta del todo con el naranja chillón y es posible que pasen desapercibidos. Tal vez en negro quedarían mejor. Y el texto podría ser un poco más oscuro para facilitar la lectura. El resto no está tan mal, podríamos definir constantes y algunos estilos en el CSS con algunos de esos colores para empezar a usarlos.

**Comment by NataliaAlvarezIspc** [comment]
Buenas! me gusta el naranja y sus distintas tonalidades (sin abusar). Podemos usar el naranja oscuro del logo para los botones y algo mas que queramos resaltar y uno mas claro para los fondos. Es cierto que si las fuentes van en negro o gris oscuro tienen una mejor lectura.

**Comment by Brian-Almada** [comment]
Quiero aportar que el naranja me gusta. Está bueno para los detalles. Como dice Nati, para botones o subrayar. También, me gusta que las letras sean en negro. Creo que jugar con el naranja (dos o tres tonalidades del mismo) y el negro está bien. No hay que cargar tanto la página de color. Creo que la parte más abarcadora del viewport (que es el fondo) debe ser blanca, sin tanta distracción. Sobre todo la sección donde se hace la subasta en sí, pues ahí la atención debe estar en los productos y en la puja. Otra cosa que me parece interesante de poner el fondo inmaculado es que podemos poner un botón para cambiar a un tema oscuro (yo soy ultrafan de eso, a todo le pongo dark mode, creo que me relaja la vista). Entonces tener un fondo uniforme, sin tanto color y detalles hace mas optimo el tema oscuro, ya que los botones y demás pueden mantener su color naranja y sólo cambiarían las fuentes a blanco.

**Comment by claubott** [comment]
Hola! Después de leer los comentarios, propongo una paleta simplificada. 

![imagen](https://user-images.githubusercontent.com/37263110/195223838-f6f4d7fe-a098-4786-be87-ae7e9e72df09.png)

En naranja del logo para títulos, íconos y botones. El naranja más oscuro únicamente para el hover de los botones. El naranja más claro solo lo uso de fondo en los inputs. Blanco y grises varios para textos y fondos.

Como ejemplo de cómo usar la paleta, agrego propuestas de modos oscuro y claro:

![imagen](https://user-images.githubusercontent.com/37263110/195224307-30065fb0-cf70-43f2-9684-6c77f4fbc9a9.png)

![imagen](https://user-images.githubusercontent.com/37263110/195224405-65c7499a-d0f3-401d-96d1-9c68cf9d865b.png)

**Comment by rpgrca** [comment]
En algunos formularios está apareciendo letra blanca en fondo naranja, eso hace que sea bastante difícil de leer. Hay que tener cuidado con eso.
![image](https://user-images.githubusercontent.com/15602473/195224652-158e45e5-2ada-4fdd-a716-25d73555b293.png)

**Reply by claubott** [reply]
qué feo! sí, varias cosas de ese css van a tener que irse.
